fuzzbuzz.yaml

```diff
@@ -0,0 +1,6 @@
+base:
+  language: go
+  docker_image: golang:latest
+  build_tags:
+    - libfuzzer
+    - blst_disabled
```

Comment:
@nisdas do we still need this tag?

Reply:
oh nvm, this is to make bls a no-op

Reply:
I think I needed this because there were a couple of places where adding the `libfuzzer` tag also necessitated it, like here https://github.com/prysmaticlabs/prysm/blob/c65c6ebc388398fd1444d09c62f7de6ea869227c/shared/bls/blst/stub.go#L1

Reply:
Makes sense thanks!